src/main/resources/assets/music_control/lang/en_us.json

```diff
@@ -56,7 +56,16 @@
     "music.volume": "Volume: %s%%",
     "music.none": "None",
 
+    "music.category.null": "None",
```

Comment:
When would this translation be triggered?

Reply:
As I don't see a use case for this, I'll remove it from this PR. If there is one, don't hesitate to tell me